component/site/controllers/icalevent.php

```diff
@@ -164,7 +164,7 @@ function editcopy(){
 
 	function save($key = NULL, $urlVar = NULL){
 		// Must be at least an event creator to save events
-		$is_event_editor = JEVHelper::isEventCreator();
+		$is_event_editor = JEVHelper::isEventEditor();
```

Comment:
This should be 
`$is_event_editor = JEVHelper::isEventCreator() || JEVHelper::isEventEditor();`
also editcopy needs changing

We should also change the doSave method of the controller (in 2 places in the method)

Reply:
@GeraintEdwards pull updated.